2021-01-18 at 16:51:25.416 [http-thread-pool::http-listener-1(4)] INFO  github.davids13.mypetfinderapp.commons.filterandinterceptors.RequestLoggingFilter.filter(RequestLoggingFilter.java:27) - Request: GET /owners
2021-01-18 at 16:52:12.893 [http-thread-pool::http-listener-1(5)] INFO  github.davids13.mypetfinderapp.commons.filterandinterceptors.RequestLoggingFilter.filter(RequestLoggingFilter.java:27) - Request: GET /owners/0
2021-01-18 at 16:52:13.073 [http-thread-pool::http-listener-1(5)] ERROR github.davids13.mypetfinderapp.commons.filterandinterceptors.ResponseLoggingFilter.checkStatus(ResponseLoggingFilter.java:46) - Response: PetErrorMessage{requestDateTime='2021.01.18 at 16:52:12 GMT', statusCode=404, petErrorCode='PET-01', petErrorDescription='Owner with this specific id=0 was not found', link='https://app.nuclino.com/illuminati-geeks/Illuminati-wrk-space/Error-Description-cf9f0f2e-7a38-4dc5-8c1e-25aae4b51fba', requestUrl='http://localhost:8080/myPetFinderApp/api/owners/0'}
